### PR TITLE
Add content tree indicator

### PR DESCRIPTION
**Pull request by msankaran0712**
## Description

Add indicator to Tree Item when expanded.
Note:
Kindly do not focus much on src/sample-workspace

Fixes # ([79](https://github.com/trimble-oss/modus-react-bootstrap/issues/79))

Preview:
https://deploy-preview-219--modus-react-bootstrap.netlify.app/components/content-tree/#Treewithindicatorforexpand

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Manual testing

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (including CHANGELOG updates)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by coliff** [comment]
I know you said to not focus on sample-workspace - but should that be included at all in this PR? Wouldn't it be better to exclude it until its ready?

File: README.md [hunk 1]
```diff
@@ -23,15 +23,23 @@ The SASS files will be found in the `/scss` directory. For further details refer
 Modus icons are required for some of the components like DataTable, FileUploadDropZone, TablePagination and TreeView. To use the icons, include the following:
 
 ```html
```
Comment (msankaran0712) [inline README.md:25]: @coliff Please review

File: src/TreeViewItem.tsx [hunk 2]
```diff
@@ -293,19 +293,19 @@ const TreeViewItem = React.forwardRef<HTMLLIElement, TreeViewItemProps>(
...
-          <TreeViewContent>
+          <ContentWrapper>
             <div
-              style={{ display: 'inline-flex' }}
+              className="d-inline-flex"
               tabIndex={finalDragIcon ? defaultTabIndex : -1}
               onClick={(e) => stopPropagation(e, !!finalDragIcon)}
               role="button"
               aria-label="Drag the item"
+              aria-hidden={!finalDragIcon}
```
Comment (msankaran0712) [inline src/TreeViewItem.tsx:303]: @coliff Now this is hidden from accessibility when the drag icon isn't present.

**Review by msankaran0712 — COMMENTED** [review]
@coliff fixed the below comments,

On the 'Tree with indicator for expand' demo there us an invisible 'Drag the item' div.

This should be removed from this example It would cause accessibility issues.
Can we also replace the inline style display: inline-flex; with class d-inline-flex.
There's an empty item-indent class. Can that be removed? Instead use utility class pl-0 ?

**Review by coliff — CHANGES_REQUESTED** [review]
this looks great. just 1 tiny last change, I think you should include `<meta name="viewport" content="width=device-width, initial-scale=1" />` meta tag in head - you can run into layout/responsive issues with it removed.

**Review by coliff — APPROVED** [review]

**Comment by msankaran0712** [comment]
Need a new PR for the latest feedback comments

**Pull request closed without merging**